packages/terra-popup/src/Popup.jsx

```diff
@@ -1,17 +1,33 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import Hookshot from 'terra-hookshot';
 import Portal from 'react-portal';
 import PopupContent from './_PopupContent';
 import PopupArrow from './_PopupArrow';
 import PopupOverlay from './_PopupOverlay';
-import TetherComponent from './_TetherComponent';
 import PopupUtils from './_PopupUtils';
-import PopupHeights from './_PopupHeights';
-import PopupWidths from './_PopupWidths';
-import './Popup.scss';
+import PopupHeights from './PopupHeights';
```

Comment:
What's the logic on removing the underscore prefix from the file name? Should we do that for the other files with underscore prefixes that imported above?

Reply:
For better consumption by more HOC, example Menu, we are exposing the height and width files as exports, and as such are removing the private denoting underscores.

Reply:
Utilities are still remaining private.

Reply:
Ok that makes sense.